setup.cfg

```diff
@@ -50,6 +50,8 @@ all =
     mlflow
     matplotlib
     tensorboardX
+    pyyaml
+    fire
```

Comment:
Hi @wyli ,

I only use `fire` in the unit tests, is it necessary to add `fire` in this `setup.cfg` to make CI pass?

Thanks.

Reply:
this is for `pip install monai[fire]`, if this pip command is required then this .cfg is correct